pygfx/materials/_points.py

```diff
@@ -216,6 +216,123 @@ class PointsGaussianBlobMaterial(PointsMaterial):
     """
 
 
+class PointsMarkerMaterial(PointsMaterial):
+    """A material to render points as markers.
+
+    Markers come in a variety of shapes, and have an edge with a separate color.
+
+    Parameters
+    ----------
+    marker : str | MarkerShape
+        The shape of the marker. Default 'circle'.
+    edge_color : str | tuple | Color
+        The color of line marking the edge of the markers. Default 'black'.
+    edge_width : float
+        The width of the edge of the markers. Default 1.
+    kwargs : Any
+        Additional kwargs will be passed to the :class:`PointsMaterial<pygfx.PointsMaterial>`.
+
+    """
+
+    uniform_type = dict(
+        PointsMaterial.uniform_type,
+        edge_color="4xf4",
+        edge_width="f4",
+    )
+
+    def __init__(self, *, marker="circle", edge_width=1, edge_color="black", **kwargs):
+        super().__init__(**kwargs)
+        self.marker = marker
+        self.edge_width = edge_width
+        self.edge_color = edge_color
+
+    @property
+    def edge_color(self):
+        """The color of the edge of the markers."""
+        return Color(self.uniform_buffer.data["edge_color"])
+
+    @edge_color.setter
+    def edge_color(self, edge_color):
+        edge_color = Color(edge_color)
+        self.uniform_buffer.data["edge_color"] = edge_color
+        self.uniform_buffer.update_range(0, 1)
+
+    @property
+    def edge_width(self):
+        """The width of the edge of the markers, in logical pixels (or world/model space if ``size_space``is set)."""
+        return float(self.uniform_buffer.data["edge_width"])
+
+    @edge_width.setter
+    def edge_width(self, edge_width):
+        self.uniform_buffer.data["edge_width"] = float(edge_width)
+        self.uniform_buffer.update_range(0, 1)
+
+    @property
+    def marker(self):
+        """The type/shape of the markers.
+
+        Supported values:
+
+        * A string from :obj:`pygfx.utils.enums.MarkerShape`.
+        * Matplotlib compatible characters: "osD+x^v<>".
+        * Unicode symbols:
+        * Emoticons:
```

Comment:
Emojis is the modern day term I believe :)

Reply:
I'm from the emoticon generation ;)